tools/enumerate-devices/rs-enumerate-devices.cpp

```diff
@@ -386,25 +403,49 @@ int main(int argc, char** argv) try
 
         if (show_modes)
         {
-            for (auto&& sensor : dev.query_sensors())
+            size_t w_res = 12;
+            size_t w_fps = 10;
+            size_t w_format = 10;
+
+            for( auto&& sensor : dev.query_sensors() )
             {
-                cout << "Stream Profiles supported by " << sensor.get_info(RS2_CAMERA_INFO_NAME) << endl;
+                cout << "Stream Profiles supported by " << sensor.get_info( RS2_CAMERA_INFO_NAME ) << endl;
+                cout << " Supported modes:\n";
 
-                cout << " Supported modes:\n" << setw(16) << "    stream" << setw(16)
-                    << " resolution" << setw(10) << " fps" << setw(10) << " format" << endl;
+                size_t w_stream = 10;
+                bool video_stream = false;
+                for( auto&& profile : sensor.get_stream_profiles() )
+                {
+                    w_stream = std::max( profile.stream_name().length(), w_stream );
+                    if( auto video = profile.as<video_stream_profile>() )
+                        video_stream = true;
+                }
+                w_stream += 2;
+
+                // Heading
+                cout << "    ";
+                if( verbose )
+                    cout << "       ";  // (sid,index)
+                cout << setw( w_stream ) << "STREAM";
+                if( video_stream )
+                    cout << setw( w_res ) << "RESOLUTION";
+                cout << setw( w_fps ) << "FPS";
+                cout << setw( w_format ) << "FORMAT";
+                cout << endl;
                 // Show which streams are supported by this device
                 for (auto&& profile : sensor.get_stream_profiles())
                 {
+                    cout << "    ";
+                    if( verbose )
+                        cout << '(' << profile.unique_id() << '.' << profile.stream_index() << ")  ";
+                    cout << setw( w_stream ) << profile.stream_name();
                     if (auto video = profile.as<video_stream_profile>())
                     {
-                        cout << "    " << profile.stream_name() << "\t  " << video.width() << "x"
-                            << video.height() << "\t@ " << profile.fps() << setw(6) << "Hz\t" << profile.format() << endl;
-                    }
-                    else
-                    {
-                        cout << "    " << profile.stream_name() << "\t N/A\t\t@ " << profile.fps()
-                            << setw(6) << "Hz\t" << profile.format() << endl;
+                        cout << setw( w_res ) << ( std::to_string( video.width() ) + 'x' + std::to_string( video.height() ));
                     }
+                    cout << setw( w_fps ) << ( "@ " + std::to_string( profile.fps() ) + "Hz" );
+                    cout << setw( w_format ) << profile.format();
+                    cout << endl;
```

Comment:
Can you share a run picture with the new changes so I can compare?

Reply:
![image](https://user-images.githubusercontent.com/54848706/200843563-a6b95dfd-a6cf-4bef-aa3f-4f3d8199e6c3.png)

Reply:
Oh got it sid, please list it in the title on verbose mode as it is not clear without reading the code..

Reply:
I don't know that it'd be much help.
It's for us -- for debugging really. I.e., if you don't know what it is, ignore it. If I put `(UID,IDX)` as a heading, will that tell you much? If you want I'll add.

Reply:
I think it's better to add something, I first thought it the USB version (2.0)